bin/run_robot

```diff
@@ -3,7 +3,7 @@
 #
```

Comment:
Yeah, that `run_robot` scriptie ideally should live in one place but we didn't update it when Willy and I did the transition to the new architecture.